testing/environments/5.0.0-cgroups.yml

```diff
@@ -0,0 +1,25 @@
+# This should test the environment with the latest snapshots
+# This is based on base.yml
+
+elasticsearch:
```

Comment:
To load all the envrionments normally the chaining of docker-compose files is used. In this case it would be `make start ENV=5.0.0-cgroups.yml`. Like this all port are already exposed and environment variables are set. In addition, this file gets much smaller (only metricbeat part needed)